### PR TITLE
Chinese translation update

### PR DESCRIPTION
**Pull request by helight**
1. fix some words， make it all translated or keep it back, such as memory, Workload, namespace.
2.translate some words yet not translated.
3.fix some translation

### CODE REVIEW

File: i18n/zh-Hans/messages.zh-Hans.xlf [hunk 1]
```diff
@@ -4954,7 +4954,7 @@
       </trans-unit>
       <trans-unit id="eeff8ad50c095fe47ae84d4ee2536b0e191c256d" datatype="html">
         <source>Session Affinity</source>
-        <target>Session Affinity</target>
```
Comment (zehuaiWANG) [inline i18n/zh-Hans/messages.zh-Hans.xlf:4957]: 这里的专有名词是不是不翻译比较好

File: i18n/zh-Hans/messages.zh-Hans.xlf [hunk 2]
```diff
@@ -2046,47 +2046,47 @@
...
-        <target>Memory 最高需求 (bytes)</target>
+        <target>内存最高需求 (bytes)</target>
         <context-group purpose="location">
           <context context-type="sourcefile">../src/app/frontend/common/components/resourcelist/node/template.html</context>
           <context context-type="linenumber">104</context>
         </context-group>
       </trans-unit>
       <trans-unit id="7d545ebc0f8d8727d50c375b2065a447f1029e2b" datatype="html">
         <source>Select namespace...</source>
-        <target>选择 namespace...</target>
```
Comment (zehuaiWANG) [inline i18n/zh-Hans/messages.zh-Hans.xlf:2065]: 看了一眼官网文档，确实已经将namespace翻译成命名空间

File: i18n/zh-Hans/messages.zh-Hans.xlf [hunk 3]
```diff
@@ -4790,7 +4790,7 @@
       </trans-unit>
       <trans-unit id="32790440573a0626970812694495678f0581d25a" datatype="html">
         <source>Volume ID</source>
-        <target>Volume ID</target>
```
Comment (zehuaiWANG) [inline i18n/zh-Hans/messages.zh-Hans.xlf:4793]: 专有名词不翻译吧，感觉卷有点奇怪

**Comment by zehuaiWANG** [comment]
/LGTM Thanks for your contribution. @helight 
Most of them look good to me. Could you help to review them again. Thanks. @hwdef @tanjunchen @chenrui333

File: i18n/zh-Hans/messages.zh-Hans.xlf (hunk 1 above)
Comment (hwdef) [reply i18n/zh-Hans/messages.zh-Hans.xlf:4957]: 会话亲和怪怪的，，感觉不翻译比较好。

File: i18n/zh-Hans/messages.zh-Hans.xlf (hunk 3 above)
Comment (helight) [reply i18n/zh-Hans/messages.zh-Hans.xlf:4793]: ok

File: i18n/zh-Hans/messages.zh-Hans.xlf (hunk 1 above)
Comment (helight) [reply i18n/zh-Hans/messages.zh-Hans.xlf:4957]: ok

File: i18n/zh-Hans/messages.zh-Hans.xlf (hunk 2 above)
Comment (helight) [reply i18n/zh-Hans/messages.zh-Hans.xlf:2065]: 是的，目前这个文件中部分翻译了，部分没有翻译。这次都统一了。

**Review by tanjunchen — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**